arch/arm64/boot/dts/qcom/sdm632-fairphone-fp3.dts

```diff
@@ -70,6 +70,15 @@
 	status = "okay";
 
 	/* Himax touchscreen @ 0x48 */
+	touchscreen@48 {
+		compatible = "himax,hx83112b";
+		reg = <0x48>;
+		interrupt-parent = <&tlmm>;
+		interrupts = <65 0x2008>;
+		touchscreen-size-x = <1080>;
+		touchscreen-size-y = <2160>;
+		reset-gpios = <&tlmm 64 0x00>;
```

Comment:
use `GPIO_ACTIVE_HIGH` instead of `0x00` but this should probably be `GPIO_ACTIVE_LOW` after fixing https://github.com/msm8953-mainline/linux/pull/43/files#r925607377

Reply:
Changed to `GPIO_ACTIVE_LOW`.